docs/profiling.md

```diff
@@ -1,22 +1,39 @@
 # Profiling
 
-## Runtime Flame Graph: `tracing` spans
+## Table of Contents
+
+- [Runtime](#runtime)
+  - [Chrome tracing format](#chrome-tracing-format)
+  - [Tracy profiler](#tracy-profiler)
+  - [Adding your own spans](#adding-your-own-spans)
+  - [Perf flame graph](#perf-flame-graph)
+- [Compile time](#compile-time)
+
+# Runtime
 
 Bevy has built-in [tracing](https://github.com/tokio-rs/tracing) spans to make it cheap and easy to profile Bevy ECS systems, render logic, engine internals, and user app code. Enable the `trace` cargo feature to enable Bevy's built-in spans.
 
 If you also want to include `wgpu` tracing spans when profiling, they are emitted at the `tracing` `info` level so you will need to make sure they are not filtered out by the `LogSettings` resource's `filter` member which defaults to `wgpu=error`. You can do this by setting the `RUST_LOG=info` environment variable when running your application.
 
-You also need to select a `tracing` backend using the following cargo features:
+You also need to select a `tracing` backend using one of the following cargo features.
+
+**⚠️ Note: for users of span based profilers**
+
+When your app is bottlenecked by the GPU, you may encounter frames that have multiple prepare-set systems all taking an unusually long time to complete, and all finishing at about the same time.
+
+Improvements are planned to resolve this issue, you can find more details in the doc comment for [`prepare_systems`](../crates/bevy_render/src/view/window/mod.rs).
```

Comment:
```suggestion
Improvements are planned to resolve this issue, you can find more details in the doc comment for [`prepare_windows`](../crates/bevy_render/src/view/window/mod.rs).
```

Also, I'm not sure I understand what is a "prepare-set" system, or what a swapchain is, what prepare_windows does, etc... Also I had to go down to "Adding your own spans" to understand what a span is, through context, so:
- maybe this doc should explain things a bit more?
- I don't think I'm qualified to review this PR ^^'

Reply:
Nice catch, included it in the fixes!

I added a link to the definition of span in `tracing`.

As for the rest, it's a bit out of scope tbh, I just wanted to add a warning to this doc, I don't know what all of those are myself yet :)